cypress/integration/table.cy.js

```diff
@@ -94,7 +95,7 @@ const programDimensions = [
 const programDataDimensions = [
     { label: TEST_DIM_AGE, value: '1991-01-01' },
```

Comment:
Does this '1991' value change? (same question for line 99)

Reply:
No for some reason the `AGE` and `DATETIME` remained static in the yearly bump, so I left them as is.